client/www/pages/docs/strong-init.md

```diff
@@ -85,3 +85,60 @@ const db = init_experimental({
   schema: schema.withRoomSchema<RoomSchema>(),
 });
 ```
+
+## Reusable types
+
+Sometimes, you'll want to abstract out your query and result types. For example, a query's result might be consumed across multiple React components, each with their own prop types. For such cases, we provide `InstantQuery` and `InstantQueryResult`.
+
+To declare a query and validate it's type against your schema, you can import `InstantQuery` and leverage [TypeScript's `satisfies` operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator) like so: `const myQuery = { myTable: {} } satisfies InstantQuery<DB>;`.
+
+To obtain the resolved result type of your query, import `InstantQueryResult` and provide it your DB and query types `const myQueryResult = InstantQueryResult<DB, typeof myQuery>`.
+
+Here's a full sample demonstranting reusable query types.
```

Comment:
Instead of including this code block here. What if we just link it to the `strong-todos` example? We do this in the [patterns docs](http://localhost:3000/docs/patterns#specify-attributes-you-want-to-query)

Reply:
sgtm.  Ideally I'd embed it as discussed, but still have to figure out a good way to do that.